src/core/core.animation.js

```diff
@@ -4,15 +4,18 @@ import helpers from '../helpers';
 
 const transparent = 'transparent';
 const interpolators = {
-	number: function(from, to, factor) {
-		return from + (to - from) * factor;
+	boolean: function(from, to, factor) {
+		return factor > 0.9 ? to : from;
```

Comment:
Why 0.9 and not 0.5?

Reply:
This is used to change the `visible` to `false`. But would probably be better to just use a suitable easing and change at 0.5